crates/broker/src/config.rs

```diff
@@ -28,11 +28,20 @@ mod defaults {
     }
 
     pub const fn lockin_gas_estimate() -> u64 {
-        1_000_000
+        // Observed cost of a lock transaction is ~135k gas.
+        // https://sepolia.etherscan.io/tx/0xe61b5cad4a45fc0913cc966f8e3ee72027c01a949a9deca916780e1245c15964
+        200_000
     }
 
     pub const fn fulfill_gas_estimate() -> u64 {
-        300_000_000
+        // Observed cost of a basic single fulfill transaction is ~350k gas.
+        // Additional padding is used to account for journals up to 10kB in size.
+        // https://sepolia.etherscan.io/tx/0x14e54fbaf0c1eda20dd0828ddd64e255ffecee4562492f8c1253b0c3f20af764
+        750_000
```

Comment:
Wow.. this is high. Would it make sense to use the results of running the executor to get the journal size for each order?

Reply:
Also is 10kb a hard limit we set somewhere?

Reply:
It's the default max size at which the prover will simply drop the request. We could try to estimate the effect of the journal size on gas costs. This number is large since its trying to be an overestimate. Maybe this is too conservative of an approach and drives up costs too much. Long-term, better gas estimation is an opportunity to compete as a prover and we expect folks to have their own way of doing it.